### PR TITLE
remove platform selector on mobile (bug 1205141)

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

**Comment by ddurst** [comment]
What, no removed with extreme prejudice? Thanks for doing this so quickly!

**Comment by ngokevin** [comment]
i'll only exercise extreme prejudice if we're removing it completely :)

**Comment by sherlockedci** [comment]
[ The Adventure of the Yellow Build](http://sherlocked.dev.mozaws.net/builds/80556194)

File: src/media/js/compat_filter.js [hunk 1]
```diff
@@ -80,11 +75,6 @@ define('compat_filter',
         views.reload();
     });
 
-    // For mobile, set limit to 10.
-    if (actualFormFactor == 'mobile' || actualPlatform == 'firefoxos') {
-        limit = 10;
```
Comment (diox) [inline src/media/js/compat_filter.js]: Wait, don't we want this ? Where else the limit to 10 on mobile would be set ?

Comment (ngokevin) [inline src/media/js/compat_filter.js]: Ah, oops. Got confused since this code was in the compat_filter file. Will restore.

**Comment by ngokevin** [comment]
Sherlocked Diff

screen shot 2015-09-16 at 11 20 27 am

**Comment by ngokevin** [comment]
Will fix tests

**Comment by sherlockedci** [comment]
[ The Adventure of the Sussex Build](http://sherlocked.dev.mozaws.net/builds/81487492)

**Comment by ngokevin** [comment]
Tests passing.

**Comment by spasovski** [comment]
Looks good locally

**Pull request merged**